src/api/expenses.controller.js

```diff
@@ -0,0 +1,89 @@
+const { expensesService } = require('../services/expenses.service.js');
+const { usersService } = require('../services/users.service.js');
+
+const getExpenses = (req, res) => {
+  const { userId, categories, from, to } = req.query;
+  const categoryList = categories ? categories.split(',') : [];
+
+  const expenses = expensesService.getExpenses({
+    userId,
+    categoryList,
+    from,
+    to,
+  });
+
+  res.json(expenses);
+};
+
+const createExpense = (req, res) => {
+  const { userId, spentAt, title, amount, category, note } = req.body;
+
+  if (!userId || !spentAt || !title || !amount || !category || !note) {
```

Comment:
Consider providing more specific error messages when returning a 400 status code. This will help clients understand which field is missing or invalid.